### DISCUSSION TITLE
CLI "Genre"

### DISCUSSION THREAD

**Discussion by protofolius** [comictagger/comictagger#537] [General]
Using the fieldname "genre" as mentioned in the wiki to masstag using CLI

`https://github.com/comictagger/comictagger/wiki/UserGuide#new-template-format`

But fails with error

`comictagger.exe: error: argument -m/--metadata: 'genre' is not a valid tag name`

Can not be used in CLI or has the name changed ?

**Comment by lordwelch** [comment]
Yes it has been changed to genres and is now a list
https://github.com/comictagger/comictagger/commit/2c3a2566cce1a871cbc3c2058a0d34ea9490dc16

**Comment by protofolius** [comment]
I was so close , yet so far .
Is there a way I could have found this information myself ?

It could be an idea to have an actual list available in the app itself , which does not need maintaining like the github wiki does ?
Or when I hover a field for a certain time it would pop up and show the field code or something ?

**Reply by lordwelch** [reply]
If you know a bit of python https://github.com/comictagger/comictagger/blob/360a9e630847771e022b556f6b966cb6f3238d02/comicapi/genericmetadata.py#L95-L160

**Reply by protofolius** [reply]
Thank you for the list . I tried using this list to find the new name for the "comments" field , but it is not to be found in the list .
Can it still be addressed ?

In the github userguide it mentions 
`{comments} | string`

But when I try to use it I get
`comictagger.exe: error: argument -m/--metadata: 'comments' is not a valid tag name`

**Reply by lordwelch** [reply]
It was renamed to description
